.github/workflows/release.yml

```diff
@@ -5,53 +5,51 @@ on:
   - workflow_dispatch
 
 jobs:
-  create_release:
-    # https://github.com/actions/create-release/issues/14#issuecomment-555379810
-    runs-on: ubuntu-20.04
-    steps:
-    - name: Create Release
-      id: create_release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: Draft Release
-        body: ''
-        draft: true
-    - name: Save Upload URL to File
-      run: printf "%s" "${{ steps.create_release.outputs.upload_url }}" > upload_url.txt
-    - name: Save Upload URL as Artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: upload_url
-        path: upload_url.txt
-
   build:
-    needs: create_release
     strategy:
       matrix:
-        os: [ubuntu20.04-aarch64, ubuntu20.04, ubuntu18.04]
+        os: [ubuntu22.04, ubuntu20.04, ubuntu18.04]
```

Comment:
We should have aarch64 builds, ideally add them back for both 20.04 and 22.04

Reply:
I may be missing something, but I don't believe there are runners for anything other than x64. To support aarch64, an additional action will have to be checked out ([example](https://github.com/uraimo/run-on-arch-action)). I can look into that if aarch64 is still desired.

Reply:
This isn't the runner OS, it's the target OS — the create-toolchain action should still run on macos-12.

Reply:
Right, it does, but the Darwin tools are compiled on each respective OS. So, without a way to compile those for aarch64, adding an aarch64 target here will result in the curl of a nonexistent Darwin tools tarball. 

I apologize if I'm missing something here. I was admittedly a bit confused how there was an aarch64 Darwin tools build in your latest release when there doesn't seem to be a mention of it anywhere in the repo's workflow or in the build script.

Reply:
mb for not clarifying, I built the aarch64 toolchain on my M1 machine — the Dockerfile in the darwin-tools repo is multi-arch. This repo doesn't face the same issue though, so aarch64 in the GH action should be okay.

Reply:
Added ubuntu22.04 (x64 and aarch64) packages to the current release, so ubuntu22.04 and ubuntu22.04-aarch64 should work now

Reply:
Re-added the aarch64 targets.